{
  "name": "eslint-plugin-no-unawaited-dot-catch-throw",
  "version": "0.0.8",
  "description": "ESLint plugin to highlight unawaited .catch() calls that contain throw statements",
  "main": "eslint-plugin-no-unawaited-dot-catch-throw.js",
  "scripts": {
    "test": "node enforce-nounawaited-dot-catch-throw.test.js",
    "publish": "npm version patch -m /\"Bumping patch version/\"",
    "preversion": "npm test",
    "build": "echo 'no build...'"
  },
  "peerDependencies": {
    "eslint": ">=7.32.0"
  },
  "keywords": [
    "eslint",
    "eslintplugin",
    "eslint-plugin",
    "catch",
    "unawaited",
    "throw"
  ],
  "author": "Reece Daniels <rubengmurray@gmail.com>",
  "license": "MIT",
  "devDependencies": {
    "eslint": "^7.32.0"
  },
  "homepage": "https://github.com/rubengmurray/no-unawaited-dot-catch-throw#readme",
  "repository": {
    "type": "git",
    "url": "https://github.com/rubengmurray/no-unawaited-dot-catch-throw.git"
  },
  "engines": {
    "node": ">=18.13.0"
  }
}
